### PR TITLE
updated frontegg libs and added account dropdown

### PR DESCRIPTION
**Pull request by aviadmizrachi**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -37,10 +37,9 @@
     "@apollo/client": "^3.2.9",
     "@frontegg/client": "^1.2.11",
     "@frontegg/react": "^0.3.23",
-    "@frontegg/react-auth": "^1.15.1",
-    "@frontegg/react-core": "^1.15.1",
-    "@frontegg/react-elements-material-ui": "^1.8.0",
-    "@frontegg/react-elements-semantic": "^1.8.0",
+    "@frontegg/react-auth": "^1.17.1",
+    "@frontegg/react-core": "^1.17.1",
+    "@frontegg/react-elements-material-ui": "^1.17.1",
```
Comment (livnekes) [inline frontend/package.json:42]: do we need this?

File: frontend/src/Components/NavBar/NavBar.scss [hunk 2]
```diff
@@ -1,5 +1,8 @@
 @import '../../Pages/MainLayout/themes.scss';
 
+.fe-account-popup{
```
Comment (livnekes) [inline frontend/src/Components/NavBar/NavBar.scss:3]: add space here

File: frontend/src/Components/NavBar/NavBar.scss [hunk 3]
```diff
@@ -65,9 +68,23 @@
       box-shadow: none;
     }
   }
-  .frontegg-notifications {
-    max-width: 50px;
+  .frontegg .wrap_notifications {
```
Comment (livnekes) [inline frontend/src/Components/NavBar/NavBar.scss:71]: replace wrap_ with wrap-

Comment (livnekes) [inline frontend/src/Components/NavBar/NavBar.scss:71]: why not using scss? 
.frontegg {
   .wrap...

File: frontend/src/Components/NavBar/NavBar.scss [hunk 4]
```diff
@@ -65,9 +68,23 @@
...
+    margin-top: 5px;
+  }
+
+  .account-wrapper {
+    display: flex;
+    justify-self: right;
+    float: right;
+    /* margin-left: 549px; */
```
Comment (livnekes) [inline frontend/src/Components/NavBar/NavBar.scss:79]: remove this

File: frontend/src/Components/NavBar/NavBar.scss [hunk 5]
```diff
@@ -65,9 +68,23 @@
...
+    position: absolute;
+    right: 50px;
+  }
+
+  .fe-account-popup{
```
Comment (livnekes) [inline frontend/src/Components/NavBar/NavBar.scss:84]: duplicate code

File: frontend/src/Components/NavBar/NavBar.scss (hunk 3 above)
Comment (aviadmizrachi) [reply frontend/src/Components/NavBar/NavBar.scss:71]: This is what we have in @frontegg/react.

Comment (livnekes) [reply frontend/src/Components/NavBar/NavBar.scss]: :(

File: frontend/src/Components/NavBar/NavBar.scss [hunk 6]
```diff
@@ -65,9 +68,20 @@
       box-shadow: none;
     }
   }
-  .frontegg-notifications {
-    max-width: 50px;
+
+  .frontegg {
+    .wrap_notifications {
+      margin-top: 5px;
```
Comment (livnekes) [inline frontend/src/Components/NavBar/NavBar.scss:74]: you miss one }

File: frontend/package.json (hunk 1 above)
Comment (livnekes) [reply frontend/package.json:42]: ?

File: frontend/src/Components/NavBar/NavBar.scss [hunk 7]
```diff
@@ -40,7 +43,9 @@
     max-width: 20rem;
     border-radius: 100px;
     border-width: 0;
-    box-sizing: border-box;
+    box-sizing: border-box;To join the video meeting, click this link: https://meet.google.com/wyx-gazr-bhv
```
Comment (livnekes) [inline frontend/src/Components/NavBar/NavBar.scss:46]: @aviadmizrachi lol  😂

**Review by livnekes — APPROVED** [review]

**Pull request merged**